stores/user/index.ts

```diff
@@ -79,29 +73,53 @@ const actions = {
   signOut
 };
 
-const reducer = (state = initialState, { type, payload }) => {
-  switch (type) {
-    case constants.SIGNING_IN:
-      return {
-        ...state,
-        signingIn: true
-      };
-    case constants.SIGNED_IN:
-      return {
-        ...state,
-        signingIn: false,
-        jwt: payload.token,
-        userId: payload.userId,
-        user: payload.user
-      };
-    case constants.SIGNING_OUT:
-      return { ...state, user: null, userId: null, jwt: null };
-    default:
-      return state;
-  }
+const initialState = {
+  userId: null,
+  jwt: null,
+  signingIn: false,
+  user: null
 };
 
+const makeUserReducer = serverState => {
+
+  const auth = serverState.apps.user;
```

Comment:
If you change `stores/index` to only pass `data.user`, then you'd have to remove `.apps` from here, too.